gotfparse/pkg/converter/converter.go

```diff
@@ -505,6 +505,10 @@ func convertCtyToNativeValue(val cty.Value) (interface{}, bool) {
 		vType = val.Type()
 	)
 
+	if val.IsMarked() {
+		return nil, true
+	}
```

Comment:
I think it could be useful to handle the `sensitive` mark specifically, and redact the value similar to how [upstream handles it](https://github.com/aquasecurity/trivy/blob/371b8cc02f2ffa3f42534a437ce8727519e7b9b9/pkg/iac/scanners/terraform/parser/funcs/redact.go#L11-L13).

```suggestion
	if val.HasMark(funcs.MarkedSensitive) {
		return "(sensitive value)", true
	}
```

This would mean a test tweak and adding an import:

```go
	"github.com/aquasecurity/trivy/pkg/iac/scanners/terraform/parser/funcs"
```

Reply:
Thanks! Applied on 5ab23fa...5ebe5ea .